### PR TITLE
feat: first iteration on player design update

### PR DESCRIPTION
**Pull request by luwes**
- default vertical design is not included yet

### CODE REVIEW

File: packages/mux-player/src/media-theme-mux/styles.css [hunk 1]
```diff
@@ -98,35 +98,34 @@ media-controller {
...
-  bottom: -2px;
-  background: linear-gradient(
-    180deg,
-    transparent,
-    transparent 3px,
-    var(--media-control-background) 3px,
-    var(--media-control-background) 8px,
-    transparent 8px,
-    transparent
-  );
```
Comment (luwes) [inline packages/mux-player/src/media-theme-mux/styles.css:118]: this was still a left over of when we made the time range bigger for getting more hover area.
it's now built-in Media Chrome so we can remove it here.

File: packages/mux-player/src/template.ts [hunk 2]
```diff
@@ -106,6 +106,7 @@ export const content = (props: MuxTemplateProps) => html`
               }
             }}"
           >
+            <svg viewBox="0 0 8 8"><circle cx="4" cy="4" r="4"></circle></svg>
```
Comment (luwes) [inline packages/mux-player/src/template.ts:109]: switched to an SVG for the dot, it's more consistent size cross-browser

File: packages/mux-player/src/index.ts [hunk 3]
```diff
@@ -817,7 +817,7 @@ class MuxPlayerElement extends VideoApiElement {
    * Get the secondary color used by the player.
    */
   get secondaryColor() {
-    return this.getAttribute(PlayerAttributes.SECONDARY_COLOR) ?? undefined;
+    return this.getAttribute(PlayerAttributes.SECONDARY_COLOR) ?? 'rgb(0 0 0 / .65)';
```
Comment (luwes) [inline packages/mux-player/src/index.ts:820]: users can still set this to an empty string to remove the secondary color

File: packages/mux-player/src/media-theme-mux/styles.css [hunk 4]
```diff
@@ -143,12 +142,26 @@ media-control-bar :is([role='button'], [role='switch'], button) {
...
+  position: relative;
+}
+
 media-controller::part(vertical-layer) {
   transition: background-color 1s;
 }
 
 media-controller:is([media-paused], :not([user-inactive]))::part(vertical-layer) {
-  background-color: var(--controls-backdrop-color, var(--controls, rgba(0, 0, 0, 0.6)));
+  background-color: var(--controls-backdrop-color, var(--controls, transparent));
```
Comment (luwes) [inline packages/mux-player/src/media-theme-mux/styles.css:164]: the drop background color can also be set back to before if the user wants to

**Review by cjpillsbury — APPROVED** [review]
Good call.

File: packages/mux-player/src/media-theme-mux/styles.css (hunk 1 above)
Comment (luwes) [reply packages/mux-player/src/media-theme-mux/styles.css:118]: this is put back in to be able to have a better experience on mobile having a higher time range bar

**Comment by cjpillsbury** [comment]
Doing some testing on the hosted app. Will add screenshots of callouts as I identify them.

**Comment by cjpillsbury** [comment]
Screen Shot 2022-10-31 at 1 29 03 PM
Live audio only could use a bit of work. 😅

**Comment by cjpillsbury** [comment]
Screen Shot 2022-10-31 at 1 33 22 PM

Live/DVR + Small shouldn't have the media-control-bar play button

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
snapshots for reference/change requests. I'll also do an audit of the code changes.

**Comment by cjpillsbury** [comment]
Screen Shot 2022-11-01 at 6 49 29 AM

Live + Extra Small has a media-time-range

**Review by cjpillsbury — APPROVED** [review]
😻

**Pull request merged**